bugsnag-android-core/src/test/java/com/bugsnag/android/JournaledDocumentStressTest.kt

```diff
@@ -0,0 +1,145 @@
+package com.bugsnag.android
+
+import com.bugsnag.android.internal.JournaledDocument
+import com.bugsnag.android.internal.JsonHelper
+import org.junit.Assert
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.TemporaryFolder
+import java.io.File
+import java.lang.Thread
+
+class JournaledDocumentStressTest {
```

Comment:
This has added on ~7s to the unit test execution time, which is more than double its current speed. Can it be dialled down a bit so that it completes in <1s?

Reply:
I've reduced the iterations by a factor of 10. Now the execution time is around 700ms.